goleveldb_test.go

```diff
@@ -43,3 +47,13 @@ func BenchmarkGoLevelDBRandomReadsWrites(b *testing.B) {
 
 	benchmarkRandomReadsWrites(b, db)
 }
+
+func TestGoLevelDBBackend(t *testing.T) {
+	name := fmt.Sprintf("test_%x", randStr(12))
+	db, err := NewDB(name, GoLevelDBBackend, "")
```

Comment:
noted that the `GoLevelDBBackend` const will always be included because it's part of `db.go` which doesn't have the `goleveldb` flag. It's not harmful because unused const are safe in Go, but in the future if we have more build flags, we could add something like

```
//go:build goleveldb
// +build goleveldb

package db

const (
	GoLevelDBBackend BackendType = "goleveldb"
)
```